docs/modules/hive/pages/index.adoc

```diff
@@ -1,7 +1,22 @@
 = Stackable Operator for Apache Hive
 
-This is an operator for Kubernetes that can manage https://hive.apache.org[Apache Hive].
-Currently, it only supports the Hive Metastore!
+This is an operator for Kubernetes that can manage https://hive.apache.org[Apache Hive] metastores.
+
+Currently it does only support running a Hive metastore service (HMS) that stores metadata about schemas and tables.
+There are multiple tools that can use the HMS:
+
+* Hive server
+** This is the "original" tool using the HMS.
+** It offers a SQL endpoint, where you can submit HiveQL (similar to SQL) queries.
+** It needs a execution engine, e.g. YARN or Spark.
+*** The reason it is currently not supported on the Stackable Platform is the complexity needed to operate YARN on kubernetes as well as the misconception of running a resource manager (YARN) on another resource manager (Kubernetes)
```

Comment:
```suggestion
*** This operator does not support running the Hive server because of  the complexity needed to operate YARN on Kubernetes. YARN is a resource manager which is not meant to be running on Kubernetes as Kubernetes already manages its own resources.
```

Reply:
I didn't want the sentence to start with "The reason it is currently not supported"

If you feel my reformulation changes too much of the original meaning, please ignore :)

Reply:
Makes sense to me, thx!